.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -59,15 +59,15 @@ body:
     attributes:
       label: Integration causing the issue
       description: >
-        The name of the integration. For example: Automation, Philips Hue
+        The name of the integration. For example: Automation, Philips Hue, ...
```

Comment:
Not married to that change, but just having the two concrete examples looked weird 😆

Reply:
What do you think?
```suggestion
        The name of the integration. For example: Automation or Philips Hue
```